WooCommerce/Classes/ViewRelated/Products/Add Product/AddProductWithAI/StartingInfo/ProductCreationAIStartingInfoView.swift

```diff
@@ -16,95 +16,117 @@ struct ProductCreationAIStartingInfoView: View {
-                        VStack(spacing: 0) {
-                            ZStack(alignment: .topLeading) {
-                                TextEditor(text: $viewModel.features)
-                                    .bodyStyle()
-                                    .foregroundStyle(.secondary)
-                                    .padding(insets: Layout.messageContentInsets)
-                                    .frame(minHeight: Layout.minimumEditorHeight, maxHeight: .infinity)
-                                    .focused($editorIsFocused)
-
-                                // Placeholder text
-                                placeholderText
-                            }
+        ScrollViewReader { proxy in
+            ScrollView {
+                VStack(alignment: .leading, spacing: Layout.parentSpacing) {
+                    VStack(alignment: .leading, spacing: Layout.titleBlockSpacing) {
+                        // Title label.
+                        Text(Localization.title)
+                            .fontWeight(.bold)
+                            .titleStyle()
+
+                        // Subtitle label.
+                        Text(Localization.subtitle)
+                            .foregroundStyle(Color(.secondaryLabel))
+                            .bodyStyle()
+                    }
 
-                            Divider()
-                                .frame(height: Layout.dividerHeight)
-                                .foregroundColor(Color(.separator))
 
-                            switch viewModel.imageState {
-                            case .empty:
-                                readTextFromPhotoButton
-                                    .padding(insets: Layout.readTextFromPhotoButtonInsets)
+                    VStack(alignment: .leading, spacing: Layout.textFieldBlockSpacing) {
+                        VStack(alignment: .leading, spacing: Layout.editorBlockSpacing) {
+                            VStack(spacing: 0) {
+                                ZStack(alignment: .topLeading) {
+                                    TextEditor(text: $viewModel.features)
+                                        .id("TextEditor")
+                                        .bodyStyle()
+                                        .foregroundStyle(.secondary)
+                                        .padding(insets: Layout.messageContentInsets)
+                                        .frame(minHeight: Layout.minimumEditorHeight, maxHeight: .infinity)
+                                        .focused($editorIsFocused)
+                                    // Scrolls to the "TextEditor" view with a smooth animation while typing.
+                                        .onChange(of: viewModel.features) { _ in
+                                            withAnimation {
+                                                proxy.scrollTo("TextEditor", anchor: .top)
+                                            }
```

Comment:
Nit: Since this is called in more than one place, maybe we should move it to a helper method for reusing?

Reply:
✔️ 6d06ec39920ec360fcdbb25cadcea7f20b5d8eae